src/ui/components/FogBackground/template.hbs

```diff
@@ -1,16 +1,4 @@
 <div data-test-fog-background>
-  <div class="intentisty" style={{opacityStyle}}>
-    <div class="fog-01">
-      <div class="image-01" />
-      <div class="image-01" />
-    </div>
-    <div class="fog-02">
-      <div class="image-02" />
-      <div class="image-02" />
-    </div>
-    <div class="fog-03">
-      <div class="image-01" />
-      <div class="image-01" />
-    </div>
-  </div>
+  <canvas id='ParticlesBackgroundCanvas' />
+  <img src='/images/fog-particle.png' id='ParticleImage' />
```

Comment:
Is this necessary to be in the template? I guess we'd want that to be loaded in the code so that we could start rendering the particles once the image is loaded?

Reply:
I had it loaded in the script before but then I thought that fingerprinting might not work properly. But I just checked and realized that images are not fingerprinted. I'll set it again to load programatically.

Reply:
So fingerprinting should work correctly when you're using the image in a script. If images are not fingerprinted that would be a problem though. Did you run a production build?